### PR TITLE
Use `molecule` composite action for the remaining 4 workflows

### LINKED ISSUE #111: the role `mirsg.infrastructure.install_python` was not found
### Short Description

Not sure when this happened, but I've noticed it via @renovatebot PRs - although I don't think responsible.

The full error
```sh
ERROR! the role 'mirsg.infrastructure.install_python' was not found in /home/runner/work/ansible-collection-infra/ansible-collection-infra/ansible_collections/mirsg/infrastructure/playbooks/molecule/resources/monitoring/roles:/home/runner/.cache/molecule/playbooks/centos7_monitoring/roles:/home/runner/work/ansible-collection-infra/ansible-collection-infra/ansible_collections/mirsg/infrastructure:/home/runner/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/home/runner/work/ansible-collection-infra/ansible-collection-infra/ansible_collections/mirsg/infrastructure/playbooks/molecule/resources/monitoring
  
  The error appears to be in '/home/runner/work/ansible-collection-infra/ansible-collection-infra/ansible_collections/mirsg/infrastructure/playbooks/molecule/resources/monitoring/converge.yml': line 7, column 7, but may
  be elsewhere in the file depending on the exact syntax problem.
  
  The offending line appears to be:
  
    roles:
      - role: mirsg.infrastructure.install_python
        ^ here
  CRITICAL Ansible return code was 1, command was: ansible-playbook --inventory /home/runner/.cache/molecule/playbooks/centos7_monitoring/inventory --skip-tags molecule-notest,notest --syntax-check /home/runner/work/ansible-collection-infra/ansible-collection-infra/ansible_collections/mirsg/infrastructure/playbooks/molecule/resources/monitoring/converge.yml
```

### Evidence/Steps to Reproduce

_No response_

### Acceptance Criteria/Expected Behaviour

_No response_

### Details

_No response_

### Resolution

_No response_

### PR DESCRIPTION
**Pull request by paddyroddy**
Follows on from #116, but made sense to be in a separate PR.

To be merged AFTER #116.

Fixes #111

### CODE REVIEW

**Comment by paddyroddy** [comment]
So we think that an issue could be due to a recent GitHub Actions runner change. We will test these on our own runner and see the affect.

**Comment by paddyroddy** [comment]
@drmatthews I've updated the workflows to use the new `ansible_major_version` following merging of https://github.com/UCL-MIRSG/.github/pull/123

**Comment by paddyroddy** [comment]
@drmatthews #116 has been merged and I've resolved conflicts here

**Comment by drmatthews** [comment]
Thanks @paddyroddy!

File: roles/omero_server/tasks/omero-systemd.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     src: systemd-system-omero-server-service.j2
     mode: "0644"
   notify:
-    - Reload systemd # handler from ome.common
+    - reload systemd # noqa: name[casing]
```
Comment (drmatthews) [inline roles/omero_server/tasks/omero-systemd.yml:12]: The handler used here is coming from the (third party) `ome.common` role.

Comment (paddyroddy) [reply roles/omero_server/tasks/omero-systemd.yml:12]: Ah that might have been my bad fighting with Ansible lint

**Review by paddyroddy — COMMENTED** [review]
I can't review, but this looks good! Does this mean `main` is fully fixed? I guess we won't know unless all actions are hit.

**Comment by drmatthews** [comment]
> Does this mean main is fully fixed?

I think it does. The only workflow not triggered this time round was the tomcat one, and that was triggered by PR 116.

**Review by HChughtai — APPROVED** [review]

**Pull request merged**